src/Helper/HeadLink.php

```diff
@@ -27,7 +26,6 @@
  * @method HeadLink prependAlternate($href, $type, $title, $extras = [])
  * @method HeadLink setAlternate($href, $type, $title, $extras = [])
  */
-// @codingStandardsIgnoreEnd
```

Comment:
This annotation is both useless and faulty:

1. Useless because it tells to ignore a docblock
1. Faulty because, placed between the docblock and the class declaration, inhibit PHP to recognize the docblock as a class phpdoc

Reply:
How we can then fix these:

![image](https://user-images.githubusercontent.com/7423207/72438025-e8001700-379b-11ea-8542-025400f5c504.png)

Reply:
I'm so sorry, I misread it as a code coverage annotation.

Now the coding standard annotation is only placed after the class declaration.